lychee.toml

```diff
@@ -0,0 +1,72 @@
+#############################  Disclaimer  #############################
+
+# Configuration file for lychee link checker.
+#
+# Further information and explanation of possible options can be found
+# at https://github.com/lycheeverse/lychee/.
+#
+# Adapted from example config file: https://github.com/lycheeverse/lychee/blob/master/lychee.example.toml
+
+#############################  Display  #############################
+
+# Verbose program output
+verbose = true
+
+#############################  Runtime  #############################
+
+# Maximum number of concurrent link checks.
+max_concurrency = 4
+
+#############################  Requests  ############################
+
+# Comma-separated list of accepted status codes for valid links.
+accept = [200, 429]
+
+# Remap URI matching pattern to different URI.
+remap = [
+    'file:///home/runner/work/music-encoding.github.io/music-encoding.github.io/_tutorials https://github.com/music-encoding/music-encoding.github.io/tree/gh-pages/tutorials/',
+    'https://github.com/music-encoding/music-encoding/tree/basic https://github.com/music-encoding/music-encoding/commit/b50748ce537b43e1527791fb41e8c28aa1582cdb',
+    'http://music-encoding.org/community/conference/tours17/program/ https://web.archive.org/web/20170511140944/http://music-encoding.org/community/conference/tours17/program',
+    'http://music-encoding.org/community/mei-organization/mei-by-laws/ https://music-encoding.org/community/mei-by-laws.html',
+    'http://music-encoding.org/proceedings https://github.com/music-encoding/mec-proceedings',
+]
+
+#############################  Exclusions  ##########################
+
+# Exclude these URLs from checking (supports regex).
+exclude = [ 
+    # custom User files
+    'file:///Users/*',
+
+    # these DOIs are not correctly resolved (probably too many redirects)
+    'https://doi.org/10.13140/RG.2.2.15014.93760',
+    'https://doi.org/10.15463/music-1',
+
+    # page outdated, but used in candidate statements
+    'https://arsmusicae.org',    
```

Comment:
Does the link checker also check spelled-out links, i.e. not those captured in cramdown link syntax? That's great!

Concerning this specific one: as it is from the 2018 candidate statements, it might be an idea to in include the whole page. See below.

Reply:
At the moment, this is the only link from alll candidate statements that doesn't work. So I wouldn't exclude all the other statements, as long as there isn't any broken link.

Reply:
Taking my own argument seriously, it could be worthy to have another look into the conferences pages, and to only exclude those specific subpages with broken links?